docs/getting-into-the-code/guidelines-for-setting-up-a-local-workspace.md

```diff
@@ -88,26 +88,27 @@ These steps are very important. They allow you to focus on the content and ensur
 
 ### Setup for IntelliJ IDEA
 
-IntelliJ IDEA fully supports Gradle as a build tool, but also has an internal build system which is usually faster. For JabRef, Gradle is required to make a full build but once set up, IntelliJ IDEA's internal system can be used for sub-sequent builds.
+We recommend to install IntelliJ IDEA using [JetBrains Toolbox App](https://www.jetbrains.com/toolbox-app/), because IDE updates are automatically installed.
```

Comment:
As these installation notes are already pretty extensive, I would try to reduce them to the minimum. How Intellj is installed is best installed is something a user can figure out themselves I think. Also the "checked for" paragraph is not really helpful for a new dev in my opinion.

Reply:
Removed the "checked for", but kept the recommendation (based on my teaching experience).

Reply:
For people who are new to the project, anything that would make them more comfortable and show them the steps towards that is great. 
Pointing to the "default" IDE, is a point that helps a lot and in JabRef cases facilitates (even if some 'mentoring' is required)